bot/exts/valentines/be_my_valentine.py

```diff
@@ -70,44 +70,38 @@ async def remove_role(self, ctx: commands.Context) -> None:
     @commands.cooldown(1, 1800, BucketType.user)
     @commands.group(name='bemyvalentine', invoke_without_command=True)
     async def send_valentine(
-        self, ctx: commands.Context, user: Optional[discord.Member] = None, *, valentine_type: str = None
+        self, ctx: commands.Context, user: discord.Member, *, valentine_type: str = None
     ) -> None:
         """
         Send a valentine to user, if specified, or to a random user with the lovefest role.
```

Comment:
Update the doc string:
```suggestion
        Send a valentine to specified user with the lovefest role.
```